xml_inscriptions/transcribed/USEP Penn Museum Amphora Handle.xml

```diff
@@ -0,0 +1,173 @@
+                            <p>stamped production label</p>
+                        </msItem>
+                    </msContents>
+
+                    <!--  *************************************  <physDesc>  ********************************* -->
+                    <physDesc>
+
+                        <!--  *************************************  <objectDesc>  *********************************  -->
+
+                        <objectDesc ana="#instrumentum.amphora">
+                            <supportDesc ana="#clay">
+                                <support>
+                                    <p> ceramic handle of Dressel Type 20/Will Type 20 amphora</p>
+                                    <dimensions type="surface" unit="cm">
+                                        <height>20.32+</height>
+                                        <width>15.88+</width>
+                                        <depth/>
+                                    </dimensions>
+                                </support>
+                                <condition ana="#fragment.single">
+                                    <p>fragment of amphora</p>
+                                </condition>
+                            </supportDesc>
+                            <!--  *************************************  <layoutDesc>  ********************************* -->
+                            <layoutDesc>
+                                <layout columns="0" writtenLines="1">
+                                    <p/>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+
+                        <!--  *************************************  <handDesc>  ********************************* -->
+
+                        <handDesc hands="1">
+                            <handNote ana="#stamped">letters incuse</handNote>
+                        </handDesc>
+
+                        <!--  *************************************  <decoDesc>  *********************************     -->
+                        <decoDesc>
+                            <decoNote>
+                                <p/>
+                            </decoNote>
+                        </decoDesc>
+                    </physDesc>
+
+                    <!--  *************************************  <history>  *********************************  -->
+
+                    <history>
+
+                        <summary>
+                            <rs/>  <!-- content can be "modern" or blank -->
+                        </summary>
+
+                        <origin>
+                            <date evidence="#archaeology" notBefore="0001" notAfter="0299">first
+                                century to third century CE</date>
+                            <placeName ref="europe.italy.rome">Monte Testaccio</placeName>
+                        </origin>
+                        <provenance type="found">
```

Comment:
As a matter of practice we normally don't give the discovery here using `<provenance>` unless there is particular detail about who found it, or specifically where it was located (if that is not already addressed by the `<origin>`.